src/ruby/yast/wfm.rb

```diff
@@ -309,6 +320,10 @@ def self.run_client(client)
       rescue SignalException => e
         handle_signal_exception(e)
         exit(16)
+      rescue AbortException => e
+        # Abort was requested
+        handle_abort_exception(e, client)
```

Comment:
do you have test? I worry that this code blown up ( two params here and method want one ).

Reply:
ups, I manually tested it just before to remove the param :(

I test it again